frontend/src/components/ElectionForm/ElectionForm.tsx

```diff
@@ -438,7 +440,7 @@ const ElectionForm = ({ authSession, onSubmitElection, prevElectionData, submitT
                         Option1={{
                             Answer: 'Yes',
                             HelpText: 'Voters will create an account and verify their email adress.',
-                            GotoPage: (process.env.REACT_APP_FF_CUSTOM_REGISTER === 'true')? 'CustomRegister?': 'Scenario8',
```

Comment:
Spotted this inconsistent naming bug as a result of using typescript features :D